src/SharedWeb/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -462,10 +462,17 @@ public static void AddNoopServices(this IServiceCollection services)
         }
 
         public static GlobalSettings AddGlobalSettingsServices(this IServiceCollection services,
-            IConfiguration configuration)
+            IConfiguration configuration, bool isDev)
```

Comment:
I don't understand what you're suggesting. I think you're saying that because we grab the `IsDevelopment` from the `IWebHostEnviornment` we should just pass that in?

Reply:
That's how we do it for the other ServiceCollectionExtension methods. We have `IWebHostEnvironment` as an argument and only use it for the `IsDevelopment` check.

Reply:
ah, fine by me